package.json

```diff
@@ -53,6 +53,11 @@
           "type": "number",
           "default": 5,
           "description": "Specify the number of minutes to wait when polling for new builds and pull requests."
+        },
+        "tfvc.location": {
+          "type": "string",
+          "default": "",
+          "description": "Specify the full path to the TF executable to use for TFVC functionality."
```

Comment:
should we change "TF executable" to  "TF executable or script"? Doesn't matter much, but since we do support tf.cmd, it might be more "correct" to include script.